src/lib/y2storage/dialogs/guided_setup.rb

```diff
@@ -60,7 +60,9 @@ def dialog_content
               VSpacing(1),
               Left(RadioButton(Id(:mode_lvm), _("LVM-based"), lvm_selected?)),
               VSpacing(1),
-              Left(RadioButton(Id(:mode_encrypted), _("Encrypted LVM-based"), encrypted_selected?))
+              Left(RadioButton(Id(:mode_encrypted), _("Encrypted LVM-based"), encrypted_selected?)),
+              VSpacing(1),
+              Left(Password(Id(:encryption_password), _("Enter encrytion password")))
```

Comment:
And the value should be initialized to `settings.encryption_password`. Otherwise this would be lost if the user re-enters the dialog.